pages/profile.js

```diff
@@ -3,187 +3,164 @@ import Link from 'next/link'
-        ],
-        moreInfoURL: null,
-      },
-      {
-        title: 'GC key password',
-        fields: ['Change my GC key password'],
-        moreInfoURL: null,
-      },
-      {
-        title: 'EI access code',
-        fields: ['Your El access code', 'XXXXXX'],
-        moreInfoURL: '#',
-      },
-    ],
-  }
+  const fakeFieldsEI = [
+    {
+      title: 'Address',
+      fields: ['123 Main street', 'Montreal (Quebec)', 'A1A 1A1'],
+      editLink: '#',
+    },
+    {
+      title: 'Province of residence',
+      fields: ['Ontario'],
+      editLink: '#',
+    },
+    {
+      title: 'Deposit Details',
+      fields: ['Scotiabank', 'XXXXXXX-123'],
+      editLink: '#',
+    },
+    {
+      title: 'Phone',
+      fields: ['XXX - XXX - 1234'],
+      editLink: '#',
+    },
+  ]
+  const fakeFieldsEI2 = [
+    {
+      title: 'Language of correspondence',
+      fields: ['English'],
+      editLink: '#',
+    },
+    {
+      title: 'Alert me',
+      fields: [
+        'Receive email notifications when new information about your claim is available',
+        'Registered',
+      ],
+      editLink: '#',
+    },
+  ]
+
+  const fakeFieldsCPP = [
+    {
+      title: 'Address',
+      fields: ['123 Main street', 'Montreal, Quebec', '1A1-1A1'],
+      editLink: '#',
+    },
+    {
+      title: 'Deposit Details',
+      fields: ['Scotiabank', 'XXXXXXX-123'],
+      editLink: '#',
+    },
+    {
+      title: 'Phone',
+      fields: ['XXX - XXX - 1234'],
+      editLink: '#',
+    },
+    {
+      title: 'Preferences',
+      fields: ['Give consent to communicate on my behalf'],
+      editLink: '#',
+    },
+  ]
+  const fakeFieldsOAS = [
+    {
+      title: 'Address',
+      fields: ['123 Main street', 'Montreal, Quebec', '1A1-1A1'],
+      editLink: '#',
+    },
+    {
+      title: 'Deposit Details',
+      fields: ['Scotiabank', 'XXXXXXX-123'],
+      editLink: '#',
+    },
+    {
+      title: 'Phone',
+      fields: ['XXX - XXX - 1234'],
+      editLink: '#',
+    },
+    {
+      title: 'Preferences',
+      fields: ['Give consent to communicate on my behalf'],
+      editLink: '#',
+    },
+  ]
+
+  const [viewCPPFields, setViewCPPFields] = useState(false)
+  const [viewEIFields, setViewEIFields] = useState(false)
+  const [viewOASFields, setViewOASFields] = useState(false)
+
+  const benefitInformations = [
+    {
+      show: viewCPPFields,
+      setShow: setViewCPPFields,
+      fields: fakeFieldsCPP,
+      benefit: t.cpp,
+    },
+    {
+      show: viewEIFields,
+      setShow: setViewEIFields,
+      fields: [...fakeFieldsEI, ...fakeFieldsEI2],
+      benefit: t.ei,
+    },
+    {
+      show: viewOASFields,
+      setShow: setViewOASFields,
+      fields: fakeFieldsOAS,
+      benefit: t.oas,
+    },
+  ]
   return (
     <>
-      <h1 className="py-4 text-4xl font-bold text-gray-darker">
-        {t.profileAndSecuritySettings}
+      <h1 className="py-4 text-4xl font-bold text-gray-darker border-b border-red-400">
```

Comment:
I didn't find this before, but we should be using the "heading" component from the design system for the h1 
https://dts-stn.github.io/DECD-Design-System/main/?path=/story/components-heading--default